scada-lts-e2e-test/scada-lts-e2e-test-impl/src/main/java/org/scadalts/e2e/test/impl/tests/service/eventdetector/EventDetectorServicePostTest.java

```diff
@@ -0,0 +1,101 @@
+package org.scadalts.e2e.test.impl.tests.service.eventdetector;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.scadalts.e2e.page.impl.criterias.*;
+import org.scadalts.e2e.page.impl.dicts.AlarmLevel;
+import org.scadalts.e2e.page.impl.dicts.DataPointType;
+import org.scadalts.e2e.page.impl.dicts.EventDetectorType;
+import org.scadalts.e2e.service.core.services.E2eResponse;
+import org.scadalts.e2e.service.impl.services.eventdetector.EventDetectorParams;
+import org.scadalts.e2e.service.impl.services.eventdetector.EventDetectorPostResponse;
+import org.scadalts.e2e.service.impl.services.eventdetector.EventDetectorResponse;
+import org.scadalts.e2e.test.impl.creators.DataSourcePointObjectsCreator;
+import org.scadalts.e2e.test.impl.runners.TestWithPageRunner;
+import org.scadalts.e2e.test.impl.utils.TestWithPageUtil;
+import org.scadalts.e2e.test.impl.utils.TestWithoutPageUtil;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(TestWithPageRunner.class)
+public class EventDetectorServicePostTest {
+
+    private String dataPointXid;
```

Comment:
I suggest you change this to Xid, instead of a string. Contains more information.